### ISSUE TITLE
Request: Plugins for PS2 and for Wii.

### ISSUE THREAD

**Issue by OPStellar** [OpenEmu/OpenEmu#4331]
I know there are various core plugins to emulate various consoles.
PCSX, PCSXR, and PCSX2 are all out of date and were never fully stable on MacOS.
Apparently, ePSXe is fully downloadable, but I can't get it to run (last updated 2016). It can apparently be installed from Homebrew as well.
I understand that OpenEmu also utilizes Homebrew in a way.
Dolphin for Mac is able to open for me, but once I try to run a Wii iso (which it is supposed to be able to handle), it just stalls out. I've had similar experience with DeSuMe.

Bottom line: OpenEmu is the most comprehensive emulator I know, and I can't get the ones it uses as cores to work on their own, so…
I have no idea how this stuff works; all I know is _that_ it _should_ be able to.
So, is it foreseeable that OpenEmu will ever be able to run PS2 or Wii ROMs?

**Comment by MaddTheSane** [comment]
1. PCSX/PCSX-R wouldn't behave any better than the Mednafen core.
2. Unless we have access to the source code of ePSXe, creating an ePSXe core is impossible.
3. Plans to implement PCSX2 are in the pipeline. Nothing is definite, though.
4. A Dolphin plug-in is already available. But it only handles GameCube games. Implementing a Wii system would require a _lot_ of work, as it uses a very complex controller that's different from anything else.

Also, what are your computer's specs?

**Comment by OPStellar** [comment]
MacBook Pro (Retina, 13-inch, Early 2015)
2.7 GHz Intel Core i5
8 GB 1867 MHz DDR3
Intel Iris Graphics 6100 1536 MB

Base model MacBook Pro from 2015
